res/Resources.php

```diff
@@ -31,14 +31,18 @@
 	'ext.smw.style' => $moduleTemplate + [
 		'styles' => [
 			'smw/ext.smw.css',
-			'smw/ext.smw.skin.css',
 			'smw/ext.smw.dropdown.css',
 			'smw/ext.smw.table.css',
 			'smw/ext.smw.tabs.css',
 			'smw/factbox/smw.factbox.css',
 			'smw/smw.indicators.css',
 			'smw/smw.jsonview.css'
 		],
+		'skinStyles' => [
+			'chameleon' => [ 'smw/ext.smw.skin-chameleon.css' ],
+			'foreground' => [ 'smw/ext.smw.skin-foreground.css' ],
+			'vector' => [ 'smw/ext.smw.skin-vector.css' ]
+		],
```

Comment:
Should all of these skin names be full lowercase? I remember there is some inconsistency with some of them starting with an uppercase letter

Reply:
Yes for skinStyles they should be in lowercase: https://codesearch.wmcloud.org/things/?q=%22skinStyles%22